webapp/src/main/webapp/src/app/app.module.ts

```diff
@@ -12,19 +11,18 @@ import { routes } from "./app.routes";
 import { RouterModule } from "@angular/router";
 import { PopoverModule } from "ngx-bootstrap/popover";
 import { Angulartics2GoogleAnalytics, Angulartics2Module } from "angulartics2";
-import { AdminModule } from "./admin/admin.module";
+import { GroupImportModule } from "./groups/import/group-import.module";
 import { GroupsModule } from "./groups/groups.module";
 
 @NgModule({
   declarations: [
     AppComponent,
-    BreadcrumbsComponent,
-    HomeComponent
+    BreadcrumbsComponent
   ],
   imports: [
     BrowserModule,
     CommonModule,
-    AdminModule,
+    GroupImportModule,
```

Comment:
Renamed the module as `AdminModule` is all encompassing so thought to target the feature area in case we have other admin functions in this app unrelated to groups.